Cargo.toml

```diff
@@ -33,6 +33,5 @@ rpath = false
 lto = true
 debug-assertions = false
 codegen-units = 1
-panic = 'abort'
```

Comment:
what ramification does this have?

Reply:
The default strategy is `unwind` which is safer than abort because it allows the application to shut down in a clean way, instead of crashing. Also the crates we're using in `ampd` require `unwind`.